components/_twig-components/functions/bem.function.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Do we need to commit this file? Or will the install scripts thing put it here for us on `npm install` or `yarn install`?